maestro-test/src/test/resources/034_press_key.yaml

```diff
@@ -3,7 +3,7 @@ appId: com.example.app
 - pressKey: Enter
 - pressKey: Backspace
 - pressKey: Home
-- pressKey: back
+- pressKey: Back
 - pressKey: Volume Up
-- pressKey: volume Down
-- pressKey: lOcK
```

Comment:
Those were actually intentional, to validate that input works regardless of the case